sources/platform/actors/running/usage_and_resources.md

```diff
@@ -42,87 +52,105 @@ Sometimes, you see the Actor's CPU use go over 100%. This is not unusual. To hel
+- Projects involving large amount of data in memory.
 
 ### Maximum memory
 
-Apify Actors are most commonly written in [Node.js](https://nodejs.org/en/), which uses a [single process thread](https://betterprogramming.pub/is-node-js-really-single-threaded-7ea59bcc8d64). Unless you use external binaries such as the Chrome browser, Puppeteer, Playwright, or other multi-threaded libraries you will not gain more CPU power from assigning your Actor more than 4 GB of memory because Node.js cannot use more than 1 core.
+Apify Actors are most commonly written in [Node.js](https://nodejs.org/en/), which uses a [single process thread](https://betterprogramming.pub/is-node-js-really-single-threaded-7ea59bcc8d64). Unless you use external binaries such as the Chrome browser, Puppeteer, Playwright, or other multi-threaded libraries you will not gain more CPU power from assigning your Actor more than `4096MB` of memory because Node.js cannot use more than 1 core.
+
+In other words, giving a [Cheerio-based crawler](https://apify.com/apify/cheerio-scraper) `16384MB` of memory (4 CPU cores) will not improve its performance, because these crawlers cannot use more than 1 CPU core.
+
+:::note Multi-threaded Node.js configuration
 
-In other words, giving a simple [Cheerio-based crawler](https://apify.com/apify/cheerio-scraper) 16GB of memory (4 CPU cores) will not make it faster because these crawlers simply cannot use more than 1 CPU core.
+It is possible to [use multiple threads in Node.js-based Actor](https://dev.to/reevranj/multiple-threads-in-nodejs-how-and-what-s-new-b23) with some configuration. This can be useful if you need to offload a part of your workload.
 
-> It is possible to [use multiple threads in Node.js-based Actor](https://dev.to/reevranj/multiple-threads-in-nodejs-how-and-what-s-new-b23) with some configuration. This can be useful if you need to offload a part of your workload.
+:::
 
 ## Usage
 
-When you run an Actor it generates platform usage that's charged towards the user account. Platform usage is composed of 4 main parts:
+When you run an Actor it generates platform usage that's charged to the user account. Platform usage comprises four main parts:
 
 - **Compute units**: CPU and memory resources consumed by the Actor.
-- **Data transfer**: Amount of data you transfered between web, Apify platform, and other external systems.
+- **Data transfer**: The amount of data transferred between the web, Apify platform, and other external systems.
 - **Proxy costs**: Residential or SERP proxy usage.
-- **Storage operations**: Read, write, and other operations towards key-value store, dataset, and request queue.
+- **Storage operations**: Read, write, and other operations performed on the Key-value store, Dataset, and Request queue.
```

Comment:
I'd add some info where to find the run usage - it's visible on run detail and in run list.